cellengine/resources/fcs_file.py

```diff
@@ -199,9 +199,15 @@ def get_events(self, inplace: bool = False, **kwargs):
 
         Returns: A DataFrame of this files data, with query parameters applied.
             If inplace=True, it updates the self.events property.
+            If destination=True, saves the file to 'destination' and returns None.
         """
 
         fresp = ce.APIClient().download_fcs_file(self.experiment_id, self._id, **kwargs)
+        if destination:
+            file = open(destination, "wb")
+            file.write(fresp)
+            file.close()
```

Comment:
This may be the one python thing I know. `with` is appropriate here, right?

```suggestion
            with open(destination, "wb") as file:
                file.write(fresp)
```

Reply:
Good catch :+1: